components/save_button.jsx

```diff
@@ -20,14 +20,20 @@ export default class SaveButton extends React.PureComponent {
 
     static defaultProps = {
         disabled: false,
-        savingMessage: localizeMessage('save_button.saving', 'Saving'),
-        defaultMessage: localizeMessage('save_button.save', 'Save'),
         btnClass: 'btn-primary',
         extraClasses: '',
     }
 
     render() {
-        const {saving, disabled, savingMessage, defaultMessage, btnClass, extraClasses, ...props} = this.props; // eslint-disable-line no-use-before-define
+        const {
```

Comment:
@lieut-data How does this component re-render when config changes? 

Regarding test as long we make sure that this component re-renders when config changes we are good. i don't think we need to test the config change reflecting new locale string as it is covered in the tests for `localizeMessage` functions

Reply:
@lieut-data Aah. We can figure this out later. seems odd but probably it was intended to trigger a re-render throughout.